sklearn/ensemble/_hist_gradient_boosting/binning.py

```diff
@@ -34,9 +34,10 @@ def _find_binning_thresholds(data, max_bins, subsample, random_state):
         If ``n_samples > subsample``, then ``sub_samples`` samples will be
         randomly chosen to compute the quantiles. If ``None``, the whole data
         is used.
-    random_state: int or numpy.random.RandomState or None
-        Pseudo-random number generator to control the random sub-sampling.
-        See :term:`random_state`.
+    random_state: int, RandomState instance or None (default)
+        Determines random number generation for random sub-sampling.
+        Use an int to make randomness deterministic.
```

Comment:
Maybe instead "use an int for reproducible results across calls"

strictly speaking, the RNG is always deterministic